backend/Dockerfile.pipelines-profile-controller

```diff
@@ -0,0 +1,8 @@
+FROM docker.io/minio/mc:RELEASE.2019-08-14T20-49-49Z as minio-cli
+
+FROM python:3.7
+# curl -o mc 'https://dl.min.io/client/mc/release/linux-amd64/archive/mc.RELEASE.2019-08-14T20-49-49Z'
+COPY --from=minio-cli /usr/bin/mc /app/mc
+RUN pip3 install 'boto3==1.22.*' 'minio==7.1.*' 'kubernetes==23.3.*'
```

Comment:
My advice is to install these python library and minio cli client during runtime. New image has to pass license review and has to be served under new registry path. I would avoid that if possible.

Reply:
I could make a runtime check and install them if they are not already there (self-built images), but we need them for the minio configuration. Another option could be to use the minio python client only. Anyway having downloads at runtime makes us dependent on the network and availability of pypi.org and the minio mc download site.